lib/util.js

```diff
@@ -240,3 +240,7 @@ util.removeTrailingSlash = function(path) {
   }
   return path;
 };
+
+util.isIE11OrLess = function(path) {
```

Comment:
Do we need `path` var?

Reply:
Also, would it make sense to pass in a version requirement instead of **only** detecting versions <= IE11?

```javascript
if (util.isSupportedIEVersion(11)) {
 // Do something
}

util.isSupportedIEVersion = function(version) {
  return !!document.documentMode && document.documentMode <= version;
}
```